src/FSharp.Stats/Testing/MultipleTesting.fs

```diff
@@ -63,6 +63,16 @@ module MultipleTesting =
         |> Seq.ofList
 
 
+    // adapted from: https://en.wikipedia.org/wiki/%C5%A0id%C3%A1k_correction   &   https://personal.utdallas.edu/~herve/Abdi-Bonferroni2007-pretty.pdf
+    // Old: /// Computes the Dunn-Šidák correction onto a collection of p-values with a given alpha. Returns a list of booleans which state if the p-value should be rejected or not, and the new critical value for this alpha.
+    /// Computes the Dunn-Šidák correction onto a collection of p-values with a given alpha. Returns a critical value. p-values above the critical value should be rejected.
```

Comment:
Try to not use special characters in fs or fsx files. Describe what this method correct for? (FWER?)